src/Compilers/CSharp/Test/Symbol/Symbols/Source/NullablePublicAPITests.cs

```diff
@@ -701,26 +699,30 @@ class C
     void M()
     {
         object? o = null;
-        _ = o;
+        var o1 = o;
         if (o == null) return;
-        _ = o;
+        var o2 = o;
     }
 }";
 
             var comp = CreateCompilation(source, options: WithNonNullTypesTrue());
 
             comp.VerifyDiagnostics();
             comp.VerifyAnalyzerDiagnostics(new[] { new NullabilityPrinter() }, null, null, true,
```

Comment:
>VerifyAnalyzerDiagnostics [](start = 17, length = 25)

Does `VerifyAnalyzerDiagnostics` generate the actual result with comments? In short, can we add the diagnostic messages here easily?

Reply:
It does not, this is the result it generates.

---
In reply to: [305071650](https://github.com/dotnet/roslyn/pull/37322#discussion_r305071650) [](ancestors = 305071650)